aggregator/src/aggregator/error.rs

```diff
@@ -146,6 +148,25 @@ pub enum Error {
     DifferentialPrivacy(VdafError),
 }
 
+/// A newtype around `Arc<Error>`. This is needed to host a customized implementation of
+/// `trillium::Handler`.
+#[derive(Clone)]
+pub(crate) struct ArcError(Arc<Error>);
```

Comment:
Is there any reason to delegate Display to the inner Error and impl Error for ArcError?

Reply:
Would a more descriptive name explain to future readers _why_ it's wrapped in an ArcError instead of just using the Error directly? Along the lines of `BatchFailureFanoutError`? I was very confused when I encountered `Arc<Error>` initially and wouldn't be any less confused about `ArcError`

Reply:
I don't think it would matter much, I only convert to this in one place (the `upload` ApiHandler function), and the codebase cares more about the Debug output of errors than Display. (This also doesn't delegate Debug, but nothing requires that)